science/ncbi_tools/Portfile

```diff
@@ -1,52 +1,79 @@
 PortSystem 1.0
 
 name            ncbi_tools
-version         20080302
+version         20120620
 description     blast is a set of tools for doing nucleotide and protein searches
 maintainers     nomaintainer
+license         public-domain
 categories      science
 platforms       darwin
 homepage        https://www.ncbi.nlm.nih.gov/blast/
 fetch.use_epsv  no
-master_sites    ftp://ftp.ncbi.nlm.nih.gov/toolbox/ncbi_tools/old/20080302
+master_sites    ftp://ftp.ncbi.nlm.nih.gov/toolbox/ncbi_tools/old/${version}
 distfiles       ncbi.tar.gz
```

Comment:
Usually this would be `distname   ncbi` and then the `.tar.gz` comes automatically from `extract.suffix`

Reply:
Fixed in the 2nd "proposed cleanup" commit